packages/i18n-calypso/src/i18n.js

```diff
@@ -291,6 +292,12 @@ I18N.prototype.setLocale = function( localeData ) {
 
 	this.state.localeSlug = this.state.locale[ '' ].localeSlug;
 
+	// extract the `textDirection` info (LTR or RTL) from either:
+	// - the translation for the special string "ltr" (standard in Core, not present in Calypso)
+	// - or the `momentjs_locale.textDirection` property present in Calypso translation files
+	this.state.textDirection =
+		this.state.locale.ltr || this.state.locale[ '' ]?.momentjs_locale?.textDirection;
```

Comment:
We actually use a context for this: https://translate.wordpress.com/projects/wpcom/-all-translated/41881/ resp.https://translate.wordpress.org/projects/wp/dev/he/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=40305&filters%5Btranslation_id%5D=1516997 
```suggestion
		this.state.locale[ 'text direction\u0004ltr' ] || this.state.locale[ '' ]?.momentjs_locale?.textDirection;
```

Reply:
Thanks @akirk! I didn't know the context is part of the translation key and that the value is an array. Fixed (I added an optional `[ 0 ]` access to your suggestion) and ready for another review!